ansible/roles/svc_dex/tasks/docker/setup_database.yml

```diff
@@ -0,0 +1,34 @@
+---
+# code: language=ansible
+# Path: ansible/roles/svc_dex/tasks/setup_docker_database.yml
+# This task file is responsible for setting up the database directory and file for the Dex service.
+
+- name: Create database directory
+  ansible.builtin.file:
+    name: "{{ dex_db_dir }}"
+    state: directory
+    mode: "777"
```

Comment:
Would a 0644 do the job rather than a 777?

Reply:
No, it's an issue between the container user and filesystem.
I plan to dig in later ( with podman and so )